### PR TITLE
feat(SubscriberHistorySentCampaigns): separate in individual component

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Change component to accept email parameter for url.
- Fix tests
- Add link in subscriber history results

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { availableLanguageOrNull } from './i18n/utils';
 import Shopify from './components/Integrations/Shopify/Shopify';
 import SignupConfirmation from './components/Signup/SignupConfirmation';
 import SubscriberHistory from './components/Reports/SubscriberHistory/SubscriberHistory';
+import SubscriberHistorySentCampaigns from './components/Reports/SubscriberHistory/SubscriberHistoryCurrentSearch/SubscriberHistorySentCampaigns/SubscriberHistorySentCampaigns';
```
Comment (cbernat) [inline src/App.js]: I'm not so sure about this, since now we can access via url, it's an independent component now, outside search. What are you thoughts?

Comment (fcoronelmakingsense) [reply src/App.js]: yes!

**Comment by cbernat** [comment]
Could you please add the link for testing? thanks!

**Comment by fcoronelmakingsense** [comment]
> Could you please add the link for testing? thanks!

http://cdn.fromdoppler.com/doppler-webapp/build2032/#/reports/campaigns-history

**Comment by cbernat** [comment]
> > Could you please add the link for testing? thanks!
> 
> http://cdn.fromdoppler.com/doppler-webapp/build2032/#/reports/campaigns-history

https://cdn.fromdoppler.com/doppler-webapp/build2032/#/reports/campaigns-history?email=cbernat@makingsense.com

**Comment by cbernat** [comment]
🚀

**Pull request merged**